operations/reservations.md

```diff
@@ -29,8 +46,14 @@ Returns all reservations from the specified interval according to the time filte
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `TimeFilter` | string [Reservation time filter](reservations.md#reservation-time-filter) | optional | Time filter of the interval. If not specified, reservations `Colliding` with the interval are returned. |
```

Comment:
I would mention here that if no filter is used, timefilter is required.

Reply:
@vitasamek check it now